src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/ReleaseControllerTests.cs

```diff
@@ -93,6 +93,7 @@ public async Task GetLatestReleaseAsSearchable()
             {
                 Id = publicationId,
                 Title = "Publication Title",
+                Summary = "<p>This is the publication summary</p>",
```

Comment:
Publication.Summary and PublicationCacheViewModel.Summary are plain text, not html.